src/Translator.js

```diff
@@ -197,7 +197,7 @@ class Translator extends EventEmitter {
       // interpolate
       let data = options.replace && typeof options.replace !== 'string' ? options.replace : options;
       if (this.options.interpolation.defaultVariables) data = { ...this.options.interpolation.defaultVariables, ...data };
-      res = this.interpolator.interpolate(res, data, options.lng || this.language);
+      res = this.interpolator.interpolate(res, data, options.lng || this.language, options);
```

Comment:
Not entirely satisfied with the additional argument here, but I also don't wish to risk breaking backwards compatibility.